openedx/core/djangoapps/user_api/models.py

```diff
@@ -27,6 +30,10 @@ class UserPreference(models.Model):
     key = models.CharField(max_length=255, db_index=True, validators=[RegexValidator(KEY_REGEX)])
     value = models.TextField()
 
+    TIME_ZONE_CHOICES = [
```

Comment:
Why is this constant defined in the user api model?  It seems generic and basic enough to be in a more common place, right?